src/components/forms/ComboBox/ComboBox.tsx

```diff
@@ -215,8 +214,8 @@ export const ComboBox = (
         value={selectedValue}
         {...selectProps}>
         {options.map((option) => (
-          <option key={option.value} value={option.value}>
-            {option.label || option.value}
+          <option key={option.id} value={option.id}>
+            {option.label || option.id}
```

Comment:
Should just be  `option.label` since label  is required now.  (same for all  other `option.label || option.id` cases)